src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -836,7 +836,7 @@ function if_include_struct_item_value(structValue, name, options)
   }
 
   if (!this.isOptional) {
-    throw new Error(`Value not provided for ${name} where one is expected`);
+    throw new Error(`Value not provided for ${name} where one is expected in ${structValue}`);
```

Comment:
Does that actually throw a useful error?  I would expect it shows "in [object Object]" or so....

Reply:
I remember hitting this error and that showed me a string. However, looking at the code above it seems like it can show `[object Object]`. Cleaned it up with a `JSON.stringify` for safety.